### PR TITLE
[service.subtitles.subtitle] 4.6.0

### PR DESCRIPTION
**Pull request by CaTzil**
### Description
* Add CAPTCH support
* Check item's media type
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [sciprt.foo.bar] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: service.subtitles.subtitle/service.py [hunk 1]
```diff
@@ -125,11 +125,13 @@ def takeTitleFromFocusedItem():
     labelTVShowTitle = xbmc.getInfoLabel("ListItem.TVShowTitle")
     labelSeason = xbmc.getInfoLabel("ListItem.Season")
     labelEpisode = xbmc.getInfoLabel("ListItem.Episode")
+    isItMovie = xbmc.getCondVisibility("Container.Content(movies)") or labelType == 'movie'
```
Comment (phil65) [inline service.subtitles.subtitle/service.py]: please use listitem.dbtype for this.

Comment (burekas7) [inline service.subtitles.subtitle/service.py]: Some of the addons doesn't set this property properly (DBTYPE) , so this condition is kind of second check in addition to listitem.dbtype.
(By the way, you already approved this code in another addon)

Comment (phil65) [inline service.subtitles.subtitle/service.py]: might be that i overlooked that, i certainly dont spot every small detail when reviewing. Feel free to keep it this way, but it is conceptually wrong and will fail in other cases. Other add-ons doing it wrong is not really a good argument imo, someone has to start doing it right. ;)

Comment (CaTzil) [inline service.subtitles.subtitle/service.py]: Can you explain what you have said about "will fail in other cases."?
What other cases?

Comment (phil65) [inline service.subtitles.subtitle/service.py]: example: link movies to a tvshow, then you get movies in a list with container.content = tvshows. ListItem.Dbtype will be correct in that case, container.content will be misleading.

Comment (CaTzil) [inline service.subtitles.subtitle/service.py]: so, do you suggest to flip the condition, so the first thing that will be checked is the  infolabel, and if that not exists check the other part?

Comment (phil65) [inline service.subtitles.subtitle/service.py]: My suggestion is to just use listitem.dbtype. Dont use container.content() at all here.

Comment (CaTzil) [inline service.subtitles.subtitle/service.py]: Ok,
I`ve reviewed the code again, and the code not affects regular usage of subtitle addon.
It only solves specific scenario for users that want to check if there is a subtitle from context menu.

Comment (CaTzil) [inline service.subtitles.subtitle/service.py]: @phil65 did you check what i`ve said?
The new code not affects regular addon usage.

**Comment by MartijnKaijser** [comment]
Improvements can be done next update

**Pull request merged**